config/config.go

```diff
@@ -103,6 +99,12 @@ type SafeConfig struct {
 	C *Config
 }
 
+func NewSafeConfig() *SafeConfig {
+	prometheus.Register(configReloadSuccess)
+	prometheus.Register(configReloadSeconds)
+	return &SafeConfig{C: &Config{}}
+}
```

Comment:
With `promauto`, this shouldn't be necessary.

Also, if you're using `prometheus.Register()`, you need to check the error return.